airsonic-main/src/main/webapp/WEB-INF/jsp/podcastChannel.jsp

```diff
@@ -70,28 +179,46 @@
             return result;
         }
 
+        function getSelectedEpisodesMediaId() {
+            var result = [];
+            for (var i = 0; i < ${fn:length(model.episodes)}; i++) {
+                var checkbox = $("#episode" + i);
+                if (checkbox.is(":checked")) {
+                    if (podcastEpisodes[i].status == 'COMPLETED') {
+                        result.push(podcastEpisodes[i].mediaFileId);
+                    } else {
+                        $().toastmessage("showErrorToast", "Download of the episode needs to be completed for that action");
```

Comment:
Is there a tool for doing the translation? What's the best method? 
Updating all the files manually sounds very exhausting.

Reply:
It is.

For now just add to at least the default locale.

Eventually we'll start using a free translation bot to update the rest of the files (weblate or something)

Reply:
needs to be put in a resource bundle...

Reply:
this function isn't necessary, and in fact, kind of is redundant with the function above, the additional filtering is only done for downloads, and you can just do that in the download block

Reply:
I'll add the default local, that's fine.

You're right, this code is kind of redundant with the code above, but there are some differences which made it complicated to avoid the redundance for my at first glance. In getSelectedEpisodes I need the value and in getSelectedEpisodesMediaId I need just the index.

But when I think twice, I think I could make one funktion with this code (jQuery to get the checked episodes), returning a JavaScript array of objects, containing the index and the the values of the checkboxes. In getSelectedEpisodes I could use the value and in getSelectedEpisodesMediaId I could just use the index.

Also the getSelectedEpisodesMediaId was used 
Actually I

Reply:
maybe:
- getSelectedEpisodes() returns an array of objects (or arrays) containing: {mediaId, id, status}
- everyone else filters this array prior to use to get the datapoints they need